docs/basic-operations.md

```diff
@@ -0,0 +1,105 @@
+
+After chosing a name, you can initialize a repository by running the `init` command:
+
+```
+$ notary init example.com/user/collection
+```
+
+On initialization, notary creates all the necessary keys needed to sign content for this collection, create the necessary TUF metadata files and push them to the remote notary server. If you already have a root key, notary uses it instead of generating a new one.
+
+The default notary server URL configured with Notary is [https://notary-server:4443/]. This default value can overridden (by priority order):
+
+	* by specifying the option `--server/-s` on commands requiring call to the notary server.
+	* by setting the `NOTARY_SERVER_URL` environment variable.
+	* by setting the `remote_server` option inside of your configuration file (defaults to `~/.notary/config.json)
+
+## Adding and deleting content from an existing trusted collection
+
+After you initialize a trusted collection you can add and remove any content from it. You can do this by running the `add` and `remove` commands, respectively:
+
+```
+$ notary add example.com/user/collection TARGET FILEPATH
+$ notary remove example.com/user/collection TARGET
+```
+
+The `add` command takes the Global Unique Name of the trusted collection, a target name, and the path to the file you want to add.
+The `remove` command takes the Global Unique Name of the trusted collection and the name of the target you want to remove.
+
+When running `add` notary calculates a cryptographic hash over the file in `FILEPATH` and point the `TARGET` name to it. Neither the `add` or `remove` commands require access to the remote notary server, since all changes are staged locally for later publishing.
+
+## Inspecting staged changes
+
+You can inspect the changes notary has pending locally for a particular trusted collection by running the `status` command:
+
+```
+$ notary status example.com/user/collection
+```
+
+These changes are kept locally until you publish this collection to the remote notary server.
+
+## Publishing changes
+
+In order for your local changes to be see by other notary clients, you have to publish them to a notary server. This can be done by using the `publish` command:
+
+```
+$ notary publish example.com/user/collection
+```
+
+When publishing, notary finds the keys necessary to sign a particular trusted collection, uses them to create a signed snapshot and uploads it to the remote notary server.
+
+## Listing targets
+
+You can list the targets that are currently part of a trusted collection by running the `list` command:
+
+```
+$ notary list example.com/user/collection
+```
+
+The `list` command will attempt to connect to the remote notary server and retrieve the most up-to-date version of the targets for a trusted collection. In case the connection fails, notary will use the locally cached metadata for up to two weeks, period after which it will start returning an error due to an expired cache.
```

Comment:
Should we make sure we distinguish the locally cached metadata from the unstaged changes?  E.g. the data has to have been published for `list` to work.